doc/news/_preparation_next_release.md

```diff
@@ -367,6 +367,14 @@ removed due to:
 - New plugin to validate hex formatted colors (e.g. #fff or #abcd) and normalize them to rgba (4294967295 (= 0xffffffff) and 2864434397 (= 0xaabbccdd) respectively). It also has support for named colors according to the [extended color keywords](https://www.w3.org/TR/css-color-3/#svg-color) from CSS3.
   _(Philipp Gackstatter)_
 
+### Ini
+
+- new formatting is introduced --> instead of having whitespaces around '=' now the whitespaces are not allowed anymore
+  example of old formatting: key1 = value1
+  example of new formatting: key1=value1
+
+      	. _(Oleksandr Shabelnyk)_
```

Comment:
Please fix the indentation of your name (exactly two spaces before the dot). Otherwise your name will show up in code style:

<img width="678" alt="Release" src="https://user-images.githubusercontent.com/691989/58725488-d3b70780-83ce-11e9-99c3-923977a987b9.png">

. If you want to know why that is the case, please take a look at the [Markdown specification](https://daringfireball.net/projects/markdown).